tests/Test.Yaapii.Xambly/Directive/XpathDirectiveTests.cs

```diff
@@ -240,60 +231,155 @@ public void NavigatesFromRootWithStrangerCursor()
 
             Assert.Contains(
                 first,
-                new XpathDirective(
-                    "/root/child"
-                ).Exec(
+                new XpathDirective("/root/child")
+                .Exec(
                     dom,
                     new DomCursor(new ManyOf<XNode>(strangerCursor)),
                     new DomStack()
                 )
             );
         }
 
-        /// <summary>
-        /// A navigator from an Xml and XPath
-        /// </summary>
-        /// <param name="xml"></param>
-        /// <param name="xpath"></param>
-        /// <returns></returns>
-        private XPathNavigator FromXPath(string xml, string xpath)
+        [Fact]
+        public void WorksWithDefaultNamespace()
         {
-            var nav =
-                new XPathDocument(
-                    new StringReader(xml)
-                ).CreateNavigator();
+            var dom = new XDocument();
+            new Xambler(
+                new Directives()
+                .Add("root")
+                    .Add("parent")
+                        .Add("child")
```

Comment:
It is not good to indent method calls to simulate an xml document. Directives are not an XML document, they are an object. It decreases readability with these indentations because it tells me there is a sub object now that there is one more indentation. Please align it all on one vertical axis.